storage/errors.go

```diff
@@ -94,15 +90,6 @@ func IsInvalidTransaction(err error) bool {
 	return false
 }
 
-// IsIndexingNotSupported returns true if this error is a IndexingNotSupportedErr.
-func IsIndexingNotSupported(err error) bool {
```

Comment:
For backwards-compatibility, we may want to keep a stub with a comment, like
```go
func IsIndexingNotSupported(error) bool { return false }
```

Reply:
Sounds good!

Reply:
Also marked deprecated.